<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectDataAnalysisElwinTab</class>
 <widget class="QWidget" name="IndirectDataAnalysisElwinTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>569</width>
         <height>232</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTabWidget" name="inputChoice">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="file">
           <attribute name="title">
            <string>File</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0">
            <item row="0" column="0">
             <widget class="MantidQt::API::FileFinderWidget" name="dsInputFiles" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>41</verstretch>
               </sizepolicy>
              </property>
              <property name="findRunFiles" stdset="0">
               <bool>false</bool>
              </property>
              <property name="label" stdset="0">
               <string>Input File</string>
              </property>
              <property name="multipleFiles" stdset="0">
               <bool>true</bool>
              </property>
              <property name="fileExtensions" stdset="0">
               <stringlist>
                <string>_red.nxs</string>
                <string>_sqw.nxs</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QFrame" name="frame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="ckGroupInput">
                 <property name="text">
                  <string>Group Input</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="ckLoadHistory">
                 <property name="text">
                  <string>Load History</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="workspace">
           <attribute name="title">
            <string>Workspace</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tbElwinData">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QFrame" name="frame_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QPushButton" name="wkspAdd">
                 <property name="text">
                  <string>Add Workspace</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="wkspRemove">
                 <property name="text">
                  <string>Remove</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>280</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>569</width>
         <height>343</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>280</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="">
           <layout class="QVBoxLayout" name="properties"/>
          </widget>
          <widget class="QWidget" name="">
           <layout class="QVBoxLayout" name="loPlotAndOptions" stretch="0,1">
            <item>
             <layout class="QHBoxLayout" name="loPreviewSelection">
              <item>
               <widget class="QLabel" name="lbPreviewFile">
                <property name="text">
                 <string>Preview file:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbPreviewFile">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbPreviewSpec">
                <property name="text">
                 <string>Spectrum:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QStackedWidget" name="elwinPreviewSpec">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="lineWidth">
                 <number>-1</number>
                </property>
                <widget class="QWidget" name="pgPlotSpinBox">
                 <widget class="QSpinBox" name="spPlotSpectrum">
                  <property name="geometry">
                   <rect>
                    <x>150</x>
                    <y>10</y>
                    <width>33</width>
                    <height>22</height>
                   </rect>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>0</number>
                  </property>
                 </widget>
                </widget>
                <widget class="QWidget" name="pgPlotCombo">
                 <widget class="QComboBox" name="cbPlotSpectrum">
                  <property name="geometry">
                   <rect>
                    <x>170</x>
                    <y>10</y>
                    <width>73</width>
                    <height>22</height>
                   </rect>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlot" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>125</height>
               </size>
              </property>
              <property name="canvasColour" stdset="0">
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="loSampleLog">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>SE log name: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leLogName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="leLogValue">
            <item>
             <property name="text">
              <string>last_value</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>average</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbPlotPreview">
            <property name="text">
             <string>Plot Current Preview</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="fResults">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="gbRun">
             <property name="title">
              <string>Run</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>7</number>
              </property>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pbRun">
                <property name="text">
                 <string>Run</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbOutput">
             <property name="title">
              <string>Output</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>7</number>
              </property>
              <item>
               <widget class="MantidQt::CustomInterfaces::IndirectPlotOptionsView" name="ipoPlotOptions" native="true"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_1">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pbSave">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Save Result</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IndirectPlotOptionsView</class>
   <extends>QWidget</extends>
   <header>IndirectPlotOptionsView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Plotting/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::API::FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/FileFinderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
